### ISSUE TITLE
download-releases-6-5.php pattern: remove unwanted alt texts

### ISSUE THREAD

**Issue by audrasjb** [WordPress/wporg-main-2022#425]
**Describe the bug**
Alt texts should be empty for illustrative images as per our Accessibility Standards: https://make.wordpress.org/accessibility/handbook/content/alternative-text-for-images/

Please remove unwanted alt texts from the `download-releases-6-5.php` pattern.

Thanks!

**Comment by ryelle** [comment]
Thanks for flagging that. I've removed all the alt-texts from that page (thinking about it like the About page, it seems like all the images are decorative). I also removed some alt text from the homepage, which also seemed decorative. See https://github.com/WordPress/wporg-main-2022/commit/9cbe881d5589249cf83de4342cdaa7dea3fecaf0

**Comment by audrasjb** [comment]
Thanks!

**Comment by bbertucc** [comment]
Curious how you determined the images were decorative @audrasjb?

I'm seeing a lot of information conveyed by the images that isn't conveyed in the text. Most notably, the image below "And Much More" includes a list of features that WordPress 6.5 introduces. I don't see any alternative text on those features.

**Comment by bbertucc** [comment]
For what it's worth, there seems to be alt text on a similar image under "See What's New" on the DotOrg home page, here: https://wordpress.org
